core/pkg/evaluator/json.go

```diff
@@ -327,6 +331,20 @@ func (je *Resolver) evaluateVariant(ctx context.Context, reqID string, flagKey s
 		metadata[SelectorMetadataKey] = selector
 	}
 
+	flagMetadata := je.store.MetadataForFlag(ctx, flag)
+	if flagMetadata.ID != "" {
+		metadata[ID] = flagMetadata.ID
+	}
+	if flagMetadata.Version != "" {
+		metadata[VERSION] = flagMetadata.Version
+	}
```

Comment:
I'm not sure what the value of these is, since we are not returning them in any responses. Maybe I'm confused.

Reply:
The idea behind this is that each flag could also have its own specific metadata. For example, a flag configuration might look like this:
```
{
  "flags": {
    "myBoolFlag": {
      "state": "ENABLED",
      "variants": {
        "on": true,
        "off": false
      },
      "defaultVariant": "on",
      "metadata": {
        "id": "sso/dev",
        "version": "1.0.0"
      }
    }
  },
  "metadata": {
    "id": "test",
    "version": "1"
  }
}
```
Here, the flag-level metadata (`id`, `version`) allows for more granular identification and versioning of individual flags, in addition to the overarching metadata. Let me know your thoughts!